portfolio/src/main/webapp/volleyball/volleyball.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Volleyball</title>
+    <link rel="stylesheet" href="volleyball.css" />
+    <link rel="stylesheet" href="/../buttons.css" />
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script>
```

Comment:
Optional nit: prefer to move script to a separate script file
https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns